trends_presentation/src/commonMain/kotlin/net/thechance/mena/trends/presentation/screen/show_real/TrendsInteractionListener.kt

```diff
@@ -0,0 +1,10 @@
+package net.thechance.mena.trends.presentation.screen.show_real
+
+interface TrendsInteractionListener {
+    fun onMoreClick()
+    fun onLikeClick(reelId: String)
+    fun onAddReelClick()
+    fun onEditTagsClick()
+    fun onManageTrendsClick()
+    fun onVideoClick(reelId: String)
```

Comment:
I think for more consistency the name will be :
```suggestion
    fun onReelClick(reelId: String)
```